packages/loader/container-definitions/src/loader.ts

```diff
@@ -125,7 +125,7 @@ export interface IContainer extends IEventProvider<IContainerEvents>, IFluidRout
     /**
      * Extract the snapshot from the detached container.
      */
-    serialize(): string;
+    serialize(): ISnapshotTree;
```

Comment:
I'd go with strings in both places, not ISnashotTree in both places.
Mostly not to expose then to internal things, be able to change it in the future (imaging different format), and avoid any potential issues with host changing accidently snapshot after rehydrating container (we delay-load things, so bits & pieces of snapshot tree are accessed way later in boot sequence)..

The only minus of strings is perf.

Reply:
Yeah. Changed to string.